data/coalition/coalition missions.txt

```diff
@@ -3211,3 +3211,604 @@ mission "Heliarchs Buy Jump Drive Mission"
+				`	(Accompany Anna during the interview.)`
+					goto "stay in interview"
+			label "directions to lounge"
+			`	Ikiyachee gives you directions to the lounge, and you head there to wait out the interview.`
+				goto lounge
+			label "accompany"
+			`	"To stay with her during the interview, we allow you to, but a lounge for you, we have. If too boring, the interview is, then leave you may."`
+			choice
+				`	(Wait in the lounge.)`
+					goto "directions to lounge"
+				`	(Accompany Anna during the interview.)`
+			label "stay in interview"
+			`	Once the first questions are asked, Anna is all too happy to begin discussing the contents of her novel in great detail.`
+			`	Under the watchful eye of the six Heliarchs in the room, Anna explains her anti-Quarg sentiments, the reasoning behind her choice to write the book, and the content of the book. She speaks quickly, but the Heliarchs' numerous questions for clarification and elaboration make the process a long one.`
+			`	"Sold many books before coming here, have you?" Temporo asks. Anna embarrassedly glances in your direction. "Well, no," she replies. "But I've been writing better and better over the past few years."`
+			`	Priyepichia's representative speaks up. "Word count, we must know. Binding process, aware of you are? You want what type, and cover style?"`
+			`	"The hardcover book should have a Jump Drive emblazoned on the front cover. I'm not sure the exact options for a binding type in this area of space; couldn't you just decide that yourself?"`
+			`	As you listen, you learn that due to the Coalition's review process to make the book more suitable, Anna will have to stay for about half a year, during which she will have an apartment to live in.`
+			choice
+				`	"Do I have to stay with Anna here for half a year?"`
+					goto "half a year"
+				`	(Say nothing.)`
+			label "rest of interview"
+			`	As the hours pass, Anna's enthusiastic tone remains strong and constant. The Kimek then takes the book, puts it in a nearby filing cabinet and thanks the two of you.`
+			`	You leave the interview room, while the Heliarchs stay, presumably to continue speaking with the Kimek. As the two of you walk out, Anna begins to speak.`
+				goto thanks
+			label "half a year"
+			`	"Required to stay with Anna, you are not, but if you wish to, stay with her you may. Return to human space, you are free to do," says the Kimek.`
+				goto "rest of interview"
+			label "lounge"
+			`	You leave the room and walk to the lounge. There are several tables where many Kimek are chatting in their native language, while a few Saryds and Arachi are visibly relaxed. You sit at an empty table to pass the time, occasionally going to the water dispenser to hydrate yourself.`
+			`	After thirty minutes, a Kimek approaches you. Their translation device speaks. "An alien in our space, you are? Most strange, this meeting truly is. Coalition member the like of you, have I never encountered. Saryd or Arachi, are you?"`
```

Comment:
Does this guy not know what a human is? I'm pretty sure that your average Coalition citizen is at least aware humans exist, and wouldn't mistake them for a Saryd or Arachi. 
```suggestion
			`	After thirty minutes, a Kimek approaches you. Their translation device speaks. "An alien in our space, you are? Most strange, this meeting truly is. A Coalition member the likes of you, have I never encountered. Saryd or Arachi, are you?"`
```